setup.py

```diff
@@ -11,7 +11,7 @@
 import sys
 import os
 
-if len(sys.argv) not in range(2, 4):
+if len(sys.argv) not in range(1, 4):
     command = ''
```

Comment:
Removed our region change? should be okay, but please comment change so if we need to deploy in different region we can